Sources/Extensions/Foundation/Array+Extensions.swift

```diff
@@ -15,3 +15,11 @@ extension Array {
         return indices.contains(index) ? self[index] : nil
     }
 }
+
+extension Array where Element: Hashable {
+    /// Return an array with any duplicates removed
+    func unique() -> [Element] {
+        var uniqueElements = Set(self)
```

Comment:
I know it's merged, but I just wanted to say that this was a smart solution to both have unique elements and also keep the order🎉